lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -350,10 +349,33 @@ class CategoryRenderer {
       percentageEl.title = Util.i18n.strings.errorLabel;
     }
 
+    if (this.hasApplicableAudits(category)) {
+      wrapper.classList.add(`lh-gauge__wrapper--${Util.calculateRating(category.score)}`);
+    } else {
+      // render gray n/a for entirely non-applicable categories
+      percentageEl.textContent = 'n/a';
+      percentageEl.style.color = '#757575';
+      percentageEl.title = Util.i18n.strings.notApplicableAuditsGroupTitle;
+    }
+
     this.dom.find('.lh-gauge__label', tmpl).textContent = category.title;
     return tmpl;
   }
 
+  /**
+   * Returns true if an LH category has any non-"notApplicable" audits.
+   * @param {LH.ReportResult.Category} category
+   * @return {Boolean}
```

Comment:
```suggestion
   * @return {boolean}
```
the typescript compiler unfortunately doesn't enforce it in js code, but it's worthwhile to make the distinction between the lowercase-b `boolean` primitive and capital-B `Boolean` object.